azure-pipelines/templates/steps/vsts-release.yml

```diff
@@ -0,0 +1,86 @@
+# File: azure-pipelines\templates\steps\nightly-snapshot-releases\vsts-release.yml
+# Description: Template to assemble and publish Snapshot/Latest versions of ${{ parameters.projectName }}
+# Variable: 'ENV_VSTS_MVN_ANDROIDADACCOUNTS_USERNAME' was defined in the Variables tab
+# Variable: 'mvnAccessToken' was defined in the Variables tab
+# Variable: 'folderName' refers to the project folder
+# Variable: 'gradleTasks' list all the gradle tasks to be used.
+# Variable: 'accessTokenName' refers to the name of the access Token in this project.
+# Variable: 'runUnitTest' boolean: refers to whether the pipeline will run unit tests or not.
+# https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate
+# Variable: 'versionType' type of version (Latest, Snapshot)
+
+parameters:
+- name: versionType
+  default: Latest
+  values:
+  - Snapshot
+  - Latest
+- name: folderName
+  default: none
+- name: projectName
+  default: none
+- name: gradleTasks
+  default: none
+- name: accessTokenName
+  default: none
+- name: runUnitTest
+  default: false
+
+jobs:
+- job: Phase_1
+  displayName: Phase 1
+  timeoutInMinutes: 90
+  cancelTimeoutInMinutes: 1
+  pool:
+    name: Hosted Windows 2019 with VS2019
+  steps:
+  - checkout: self
+    clean: true
+    submodules: recursive
+    persistCredentials: True
+  - template: credscan-policheck.yml
```

Comment:
do not use the credscan-policheck.yml this will be checked separately
check https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1437